agent/src/main/java/reactor/blockhound/BlockHound.java

```diff
@@ -230,7 +317,9 @@ public void install() {
                 threadPredicate.test(Thread.currentThread());
                 BlockHoundRuntime.threadPredicate = threadPredicate;
 
+                java.time.Instant before = java.time.Instant.now();
                 instrument(instrumentation);
+                System.out.println("Instrumentation took  " + java.time.Duration.between(before, java.time.Instant.now()));
```

Comment:
would it be possible to use a logger here instead, so that this debug info can be suppressed? If I'm not mistaken the `Logger` / `LoggerFactory` parts are independent enough of the instrumented part that it wouldn't trigger classloading issues.

alternatively, consider returning the `Duration` in the future (which shouldn't be a breaking change)

Reply:
oooops! This change wasn't supported to be pushed at all :D Thanks for spotting, will revert 👍